packages/component-survey/.gitignore

```diff
@@ -0,0 +1,6 @@
+doc/
+dist/
+src/generated/
+.eslintcache
+*.orig
+old/
```

Comment:
what generates `old` folders?  or `.orig` files?

Reply:
That's left in from some munging I did once, I'll remove them